dev/TabView/TabView.xaml

```diff
@@ -112,6 +70,62 @@
         </Setter>
     </Style>
 
+    <Style x:Name="TabViewListViewStyle" TargetType="ListView">
```

Comment:
The IDL had a custom ListView type, so shouldn't this be the default style for TabViewListView?

Reply:
yes, it should!